### DISCUSSION TITLE
superset能否通过API传入数据动态生成图表的问题？请各位大哥大姐帮忙！！！！谢谢

### DISCUSSION THREAD

**Discussion by 543645165** [apache/superset#24780] [Q&A / Help]
背景：公司要使用一个bi工具（选了superset），目前需要动态生成图标，主要流程为：通过API上传sql查数据（或者直接上传生成图表的全部数据），然后生成图表，生成图表页面嵌套在自己公司开发的项目上！
各位大哥大姐有类似的开发经历不，如果有的话帮忙指点一下小弟！！！！谢谢！！！（官网的swagger API文档接口有很多，但是没啥注释，有点不知道用哪些接口）

**Comment by zuaa** [comment]
@543645165  找到方案了吗？知道是哪个接口生成快照吗？？
